grails-app/init/au/org/ala/biocache/hubs/BiocacheHubsBootStrap.groovy

```diff
@@ -0,0 +1,8 @@
+package au.org.ala.biocache.hubs
```

Comment:
it's removed.

Reply:
Turns out there is a version in there already so i was wrong - see https://github.com/AtlasOfLivingAustralia/biocache-hubs/blob/develop/grails-app/init/biocache/hubs/BootStrap.groovy

Reply:
This file (named `BiocacheHubsBootStrap.groovy` was deleted since dq feature doesn't need it.

What vrvj created is `BootStrap.groovy`, it will still be there.